woocommerce/woocommerce
# Fix release post template syntax errors and add prettierignore

### All Submissions:






### Changes proposed in this Pull Request:

* Fix some templates where we forgot to close block tags and it causes "block recovery" errors.
* Add a prettierignore. We don't have prettier setup explicitly in here just yet, but if you do have it set up it tries to format ejs files which is incorrect because prettier does not support EJS.






### How to test the changes in this Pull Request:



1. Generate a release post e.g. with `pnpm run release-post -- "7.3.0" --outputOnly`
2. Copy the output and paste into WP block editor
3. None of the output should generate "block recovery errors"



### Other information:




### FOR PR REVIEWER ONLY:
